wiki/matomo.js

```diff
@@ -1,13 +1,15 @@
-var _paq = window._paq || [];
+var _paq = window._paq = window._paq || [];
 /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
 _paq.push(['trackPageView']);
 _paq.push(['enableLinkTracking']);
 (function() {
-    var u="//cloud.denbi.de/matomo/";
-    _paq.push(['setTrackerUrl', u+'matomo.php']);
-    _paq.push(['setSiteId', '2']);
-    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
-    g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'matomo.js'; s.parentNode.insertBefore(g,s);
-})();
-
-
+    var u = "https://piwik.cebitec.uni-bielefeld.de/";
+    _paq.push(['setTrackerUrl', u + 'matomo.php']);
+    _paq.push(['setSiteId', '22']);
```

Comment:
Does it have to be hardcoded or could we also define it somewhere in the evn or something?